products/slackware.md

```diff
@@ -37,7 +36,7 @@ releases:
     releaseDate: 2013-11-04
 -   releaseCycle: "14.0"
     eol: false
-    releaseDate: 20l2-09-26
+    releaseDate: 2012-09-26
```

Comment:
@alexkoster  The validation didn't run on your PR due to GitHub restrictions, but it caught this on my PR. Did you copy this from somewhere by any chance? Would be nice to correct it upstream as well in that case

Reply:
@captn3m0 Good catch! That must have been a typo on my part.  AFAIK I did not copy it from anywhere since I was referencing the following text from [this chagenlog](ftp://mirrors.syringanetworks.net/slackware/slackware-14.0/ChangeLog.txt): 

`+--------------------------+
Wed Sep 26 01:10:42 UTC 2012
Slackware 14.0 x86 stable is released!
`